### PR TITLE
Add AssetIcon, Button, ButtonsGroup, Hint

### PR DESCRIPTION
**Pull request by winston0410**
Add AssetIcon, Button, ButtonsGroup, Hint

### CODE REVIEW

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Chromatic failing
Screenshot 2022-02-04 at 15 41 02

File: src/components/ui/ButtonsGroup/ButtonsGroup.test.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import ButtonsGroup from './index';
+
+describe('ButtonsGroup', () => {
+  it('renders with left text and right text', () => {
+    render(
+      <ButtonsGroup
+        leftText={'foo'}
```
Comment (vadim-kirpun) [inline src/components/ui/ButtonsGroup/ButtonsGroup.test.tsx:8]: Let's format it and remove curly braces

File: src/components/ui/Hint/Hint.test.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+
+import Hint from './index';
+
+describe('Hint', () => {
+  it('renders with given text', () => {
+    render(<Hint text={'foo'} iconType={'warning'} />);
```
Comment (vadim-kirpun) [inline src/components/ui/Hint/Hint.test.tsx:7]: Let's remove curly braces

File: src/components/ui/Hint/Hint.test.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    const content = screen.getByText('foo');
+
+    expect(content).toBeInTheDocument();
+  });
+
+  it('renders with correct icon', () => {
+    render(<Hint text={'foo'} iconType={'warning'} />);
```
Comment (vadim-kirpun) [inline src/components/ui/Hint/Hint.test.tsx:15]: Let's remove curly braces

**Review by vadim-kirpun — CHANGES_REQUESTED** [review]

File: src/components/ui/ButtonsGroup/ButtonsGroup.test.tsx (hunk 1 above)
Comment (winston0410) [reply src/components/ui/ButtonsGroup/ButtonsGroup.test.tsx:8]: Prettier is not stripping off these braces automatically. I advise we don't change this now or change prettier config

**Review by vadim-kirpun — APPROVED** [review]

**Pull request merged**